docs/api.md

```diff
@@ -7,26 +7,144 @@ Loosely based off the example in the README
 
 | Name | Type | Description | Example |
 | ------- | ------- | ------- | ------- |
-| **[black_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
-| **date_played** | *date-time* |  | `"2015-01-01T12:00:00Z"` |
-| **game_record** | *string* |  | `"<raw sgf string>"` |
-| **game_server** | *string* |  | `"KGS"` |
-| **rated** | *boolean* |  | `true` |
-| **[white_id](#resource-player)** | *number* | this could be a uuid to make it db agnostic | `42.0` |
+| **[black_id](#resource-player)** | *integer* | black player's id | `42` |
+| **date_played** | *date-time* | time the game was played | `"2015-01-01T12:00:00Z"` |
```

Comment:
That's a `prmd` thing.  This is a generated file.  I can put it in the header if you like.